app/lib/check_elastic_search.rb

```diff
@@ -63,7 +63,7 @@ def update_elasticsearch_doc_bg(options)
       create_doc_if_not_exists(options)
       sleep 1
       client = $repository.client
-      client.update index: CheckElasticSearchModel.get_index_alias, id: options[:doc_id], retry_on_conflict: 3, body: { doc: fields }
+      client.update index: CheckElasticSearchModel.get_index_alias, id: options[:doc_id], body: { doc: fields }
```

Comment:
Question: Did you confirm that the default is zero? So we don't need to set it explicitly?

Reply:
Yes it's optional and the default is zero.